modules/axruntime/src/mp.rs

```diff
@@ -8,10 +8,11 @@ static mut SECONDARY_BOOT_STACK: [[u8; TASK_STACK_SIZE]; SMP - 1] = [[0; TASK_ST
 
 static ENTERED_CPUS: AtomicUsize = AtomicUsize::new(1);
 
+#[allow(clippy::absurd_extreme_comparisons)]
```

Comment:
Why we need this? Could you paste the build warnings here?

Reply:
Because default SMP is 1, other targets will be 1 - 1 and never run.

Reply:
```shell
error: this comparison involving the minimum or maximum element for this type contains a case that is always true or always false
  --> modules\axruntime\src\mp.rs:15:35
   |
15 |         if i != primary_cpu_id && logic_cpu_id < SMP - 1 {
   |                                   ^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: because `SMP - 1` is the minimum value for this type, this comparison is always false
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#absurd_extreme_comparisons
   = note: `#[deny(clippy::absurd_extreme_comparisons)]` on by default
```

Reply:
Maybe we can add a judge statement before the loop like below:
```rust
pub fn start_secondary_cpus(primary_cpu_id: usize) {
    if SMP <= 1 {
         return;
    }
    let mut logic_cpu_id = 0;
    for i in 0..SMP {
        ...
        login_cpu_id += 1;
    }
}
```
Then we no longer need to compare `logic_cpu_id` with `SMP` and can remove this attribute.

Reply:
This judgment is used to prevent all checks of `i != primary_cpu_id` is passed and causing `logic_cpu_id` to overflow, is it right? @ZR233

Reply:
> This judgment is used to prevent all checks of `i != primary_cpu_id` is passed and causing `logic_cpu_id` to overflow, is it right? @ZR233

oh, you're right, my bad.